sdk/communication/azure-communication-administration/src/main/java/com/azure/communication/administration/implementation/package-info.java

```diff
@@ -2,8 +2,5 @@
 // Licensed under the MIT License.
 // Code generated by Microsoft (R) AutoRest Code Generator.
 
-/**
- * Package containing the implementations and inner classes for CommunicationIdentityClient. Azure Communication
- * Identity Service.
- */
+/** Package containing the implementations for CommunicationIdentityClient. Azure Communication Identity Service. */
```

Comment:
The administration package should not have references to CommunicationIdentityClient anymore. Let's make sure this is removed.

Reply:
update the comment